jest.config.js

```diff
@@ -8,4 +8,7 @@ module.exports = {
       lines: 90,
     },
   },
+  globals: {
+    structuredClone: {}
```

Comment:
The trouble seems to be that node has defined `structuredClone` in v19 and v20, and jest-environment-node is polyfilling it improperly. We don't use the function anywhere, so we can define it ourselves and it'll work. I expect we'll see a fix in the future to jest-environment-node that'll make it so we don't have to include this line -- maybe we should add an issue or something to follow up on this in the future?

Reply:
Let's go ahead and make an issue so we have it on our backlog somewhere

Reply:
Created #32